videoframes.py

```diff
@@ -158,6 +158,10 @@ def setup_ui(self):
         self.findChild(QtCore.QObject, "delete_button").clicked.connect(self.delete_stream)
         self.findChild(QtCore.QObject, "pause_button").clicked.connect(self.toggle_playback)
 
+        self.delete_button.hide()
```

Comment:
Instead of hiding these here we perhaps should use a property in the ui files? As the default is hidden.